code/datums/elements/turf_transparency.dm

```diff
@@ -1,5 +1,6 @@
 
 /datum/element/turf_z_transparency
+	element_flags = ELEMENT_DETACH
```

Comment:
my bad but it'd be nice if we checked for an implemented /detach proc without the flag maybe?

Reply:
it's only really needed here because turf signals persist now. otherwise there'd be no reason to detach on destroy.

Reply:
@LemonInTheDark my point is that this element implemented the detach proc, but for some reason did not have the flag, if thats the case we should know because either:
1. the detach proc shouldn't be overriden
2. the flag should be set

Reply:
well it's ok for elements to implement detach() without the flag, since the flag indicates that they want to remove themselves on parent destroy, whereas detach() can be called by manually removing the element